### ISSUE TITLE
OpenCV + OpenCL is greyed out when they are installed.

### ISSUE THREAD

**Issue by evertvorster** [slowmoVideo/slowmoVideo#193]
Hi there!
It's been many years since I have played with this software, and I am glad to see that you still tinker with it from time to time. 
On Arch Linux, slowmovideo does not seem to detect OpenCL, even though it's installed. (The menu selection for this is greyed out)
clinfo returns 1 platform with nVidia CUDA. (and a very long list of supported stuff)
OpenCV is at version 4.5.2

Is OpenCV + OpenCL supposed to be detected at compile time? Is it a switch I need to turn on in compile?
Or does it just have to be present on the system?

**Comment by schranzonator** [comment]
Yes and can it use more than 1 cpu? It renders forever with just 15% processor usage ^^
